pymc3/distributions/mixture.py

```diff
@@ -197,21 +212,21 @@ class NormalMixture(Mixture):
         the component standard deviations
     tau : array of floats
         the component precisions
+    distshape : shape of the Normal component
+        notice that it should be different than the shape
+        of the mixture distribution, with one axis being
+        the number of component.
```

Comment:
small typo: "number of component**s**".  I am also not sure I understand the docstring either -- would adding "a mixture of three 2d normal distributions would have `shape=(3, 2)` and `dist_shape=(2,)`" be accurate?

Reply:
Yeah this is a tricky one.
"a mixture of three 2d normal distributions would have `shape=(3, 2)` and `dist_shape=(2,)`" this is not correct. A mixture of three 2d normal distributions would have `shape=(..., 2)`, and `dist_shape=(2, 3)`. But I am not sure multi-D normal mixture actually works... 
In practice, if you have a NormalMixture RV that is `shape=(a,b,c,...)`, the component shape get one more axis `comp_shape=(a,b,c,...,k)`, with `k` components. However, passing `comp_shape=(a,b,c,...,k)` (also what the previous code trying to do) break cases when you are using `theano.shared` observed and sample_ppc.